src/js/angular/rest/graph-data.rest.service.js

```diff
@@ -136,4 +153,32 @@ function GraphDataRestService($http) {
             }
         });
     }
+
+    function resolveGraphs(scope, selectedGraph) {
+        if (scope.getActiveRepository() !== "") {
+            scope.graphsInRepo = [];
+            return RDF4JRepositoriesRestService.getGraphs(scope.getActiveRepository())
+                .success(function (graphs) {
+                    graphs.results.bindings.unshift({
+                        contextID: {
+                            type: "default",
+                            value: "The default graph"
+                        }
+                    });
+
+                    Object.keys(graphs.results.bindings).forEach(function (key) {
+                        const binding = graphs.results.bindings[key];
+                        if (binding.contextID.type === "bnode") {
+                            binding.contextID.value = '_:' + binding.contextID.value;
+                        } else if (binding.contextID.type === "default") {
+                            binding.contextID.uri = "http://www.openrdf.org/schema/sesame#nil";
+                        } else {
+                            binding.contextID.uri = binding.contextID.value;
+                        }
+                    });
+                    graphs.results.bindings.unshift(selectedGraph);
+                    scope.graphsInRepo = graphs.results.bindings;
```

Comment:
And this should not happen as well `scope.graphsInRepo = graphs.results.bindings;`. This can be the return value and the scope.graphsInRepo assignment to happen in the controller.